src/users/views.py

```diff
@@ -75,17 +82,40 @@ def post(self, request):
             last_name=request.data.get('last_name', ''),
         )
 
+        check = re.search(r'[\w.]+\@(g.)?ucla.edu', new_user.email)
+        if check is None:
+            raise Exception("Invalid Email")
+
         new_profile = Profile(
             user=new_user,
+            verification_code=''.join(random.choices(string.ascii_uppercase+string.digits, k=Profile.VERIFICATION_CHAR_NUM)),
```

Comment:
can we make this a profile static method?  I feel like it doesn't really belong here.